helpme/tests.py

```diff
@@ -1,3 +1,270 @@
-from django.test import TestCase
+from django.test import TestCase, Client
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Permission
+from django.contrib.sites.models import Site
+from django.urls import reverse
 
-# Create your tests here.
+from .models import Ticket, Comment, Team, StatusChoices, PriorityChoices, VisibilityChoices
+from .settings import app_settings
+
+class ModelTests(TestCase):
+
+    def test_create_ticket_model(self):
+        ticket = Ticket.objects.create(subject="Test", description="Test description", category=app_settings.TICKET_CATEGORIES.HELP)
+        
+        self.assertEqual(ticket.site.domain, "example.com")
+        self.assertEqual(ticket.status, StatusChoices.OPEN)
+        self.assertEqual(ticket.priority, PriorityChoices.MEDIUM)
+        self.assertEqual(ticket.category, app_settings.TICKET_CATEGORIES.HELP)
+        self.assertEqual(ticket.subject, "Test")
+        self.assertEqual(ticket.description, "Test description")
+
+        
+    def test_create_comment_model(self):
+        ticket = Ticket.objects.create(subject="Test", description="Test description", category=app_settings.TICKET_CATEGORIES.HELP)
+        comment = Comment.objects.create(content="This is an example comment", ticket=ticket)
+    
+        self.assertEqual(comment.visibility, VisibilityChoices.REPORTERS)
+        self.assertEqual(comment.content, "This is an example comment")
+        self.assertIs(comment.ticket, ticket)
+
+        
+    def test_create_team_model(self):
+        team = Team.objects.create(name="Test Team", global_team=True, categories=str(app_settings.TICKET_CATEGORIES.COMMENT) + "," + str(app_settings.TICKET_CATEGORIES.HELP))
+        
+        self.assertEqual(team.categories, "1,3")
+        self.assertEqual(team.name, "Test Team")
+        self.assertIs(team.global_team, True)
+
+        
+class ClientTests(TestCase):
+
+    @classmethod
+    def setUpTestData(cls):
```

Comment:
I usually do this in the setUp() method (which you have after).  How is doing it this way that is different/better?  Curiosity rather than criticism.